### ISSUE TITLE
mubi.com support

### ISSUE THREAD

**Issue by ghost** [ytdl-org/youtube-dl#16091]
## Please follow the guide below

- You will be asked some questions and requested to provide some information, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *issue* (like this: `[x]`)
- Use the *Preview* tab to see what your issue will actually look like

---

### Make sure you are using the *latest* version: run `youtube-dl --version` and ensure your version is *2018.04.03*. If it's not, read [this FAQ entry](https://github.com/rg3/youtube-dl/blob/master/README.md#how-do-i-update-youtube-dl) and update. Issues with outdated version will be rejected.
- [x] I've **verified** and **I assure** that I'm running youtube-dl **2018.04.03**

### Before submitting an *issue* make sure you have:
- [x] At least skimmed through the [README](https://github.com/rg3/youtube-dl/blob/master/README.md), **most notably** the [FAQ](https://github.com/rg3/youtube-dl#faq) and [BUGS](https://github.com/rg3/youtube-dl#bugs) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?type=Issues) the bugtracker for similar issues including closed ones
- [x] Checked that provided video/audio/playlist URLs (if any) are alive and playable in a browser

### What is the purpose of your *issue*?
- [ ] Bug report (encountered problems with youtube-dl)
- [x] Site support request (request for adding support for a new site)
- [ ] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other

---

### If the purpose of this *issue* is a *site support request* please provide all kinds of example URLs support for which should be included (replace following example URLs by **yours**):
- Single video: https://mubi.com/showing/the-comedy (in Netherlands, the selection of films is dependent on location)
- Playlist: https://mubi.com/lists/my-100-personal-favorites (get those that are currently available)
Note that **youtube-dl does not support sites dedicated to [copyright infringement](https://github.com/rg3/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free)**. In order for site support request to be accepted all provided example URLs should not violate any copyrights.

---

### Description of your *issue*, suggested solution and other information

[There is already working scraper for Kodi/XBMC](https://github.com/jamieu/plugin.video.mubi). As far as I see most videos use dash and also there are some films with DRM, but as the creator of the plugin said, that is minority.

**Comment by tomryanx** [comment]
+1

**Comment by KRE02** [comment]
It seems Mubi has rolled out DRM for all their films.

**Comment by maricn** [comment]
So far it was working when you inspect element and get the mpd playlist file, but now they introduced the DRM protection.. I wouldn't mind it really, only if there was an option to use custom subtitle files ..

**Comment by 89z** [comment]
Issue should be closed, URL is dead

**Comment by gamer191** [comment]
Actually, it should be closed as a duplicate of #23899

**Comment by 3052** [comment]
its possible to download from Mubi, just not with YouTube-DL/YT-DLP because of policy reasons:

~~~
15:03:01 INFO GET URL=https://dallas-edge3.mubicdn.net/stream/d8139004d0d251f0711c94d067eb140d/671f0ce7/48f44f20/mubi-films/8696/the-fall_eng_zxx_3840x2076_80000_mezz45879/34c03414fa/drm_playlist.68811912a1.ism/default/ver1.AVC1.2160p.ex-eac3.mpd
15:03:01 INFO GET URL=https://api.mubi.com/v3/films/the-fall
15:03:01 INFO GET URL="https://dallas-edge3.mubicdn.net/stream/d8139004d0d251f0711c94d067eb140d/671f0ce7/48f44f20/mubi-films/8696/the-fall_eng_zxx_3840x2076_80000_mezz45879/34c03414fa/drm_playlist.68811912a1.ism/default/dash/drm_playlist.68811912a1-video=5500047.dash"
15:03:02 INFO POST URL=https://lic.drmtoday.com/license-proxy-widevine/cenc/
15:03:02 INFO CDM PSSH=CAESENGANnriJEuQkzSK/yUR1HYaCHVzcC1jZW5jIhgwWUEyZXVJa1M1Q1ROSXIvSlJIVWRnPT0qADIA key="14AZvuC7KynTaxzd0chenQ=="
15:03:03 INFO 0.53 % size="19.93 megabyte" rate="19.59 megabyte/s"
~~~

**Comment by dirkf** [comment]
Anyhow, this should have been closed with #23899 as pointed out above.
